src/main/java/core/basesyntax/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.strategy.OperationStrategy;
+
+public class TransactionServiceImpl implements TransactionService {
+    private static final OperationStrategy OPERATION_STRATEGY =
+            new OperationStrategyImpl(new OperationHandlerProvider().getOperationHandlers());
```

Comment:
```suggestion
    private final OperationStrategy operationStrategy =
            new OperationStrategyImpl(new OperationHandlerProvider().getOperationHandlers());
```
this is not a magic number or string, just a field